internal/bminventory/inventory_test.go

```diff
@@ -5133,6 +5133,37 @@ var _ = Describe("[V2ClusterUpdate] cluster", func() {
 					Expect(*actual.Platform.Type).To(Equal(models.PlatformTypeVsphere))
 				})
 
+				It("Update UMN=true while cluster platform already set to vsphere - success", func() {
```

Comment:
Please add the following tests:
1. cluster platform none and an update request that set the cluster platform to vsphere + UMN=true
2. cluster platform none and an update request that set the cluster platform to vsphere + UMN=false
3. cluster platform baremetal and an update request that set the cluster platform to vsphere + UMN=false
4. cluster platform baremetal and an update request that set the cluster platform to vsphere + UMN=true

Reply:
Done